addons/binding/org.openhab.binding.tankerkoenig/OSGI-INF/TankerkoenigHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
I did get lots of error running this command. I changed the header of that file manualy.